aspnet/web-api/overview/advanced/calling-a-web-api-from-a-net-client.md

```diff
@@ -29,106 +29,141 @@ In this tutorial, a client app is written that consumes the following web API.
-
+To learn how to implement this API with ASP.NET Web API, see [Creating a Web API that Supports CRUD Operations](xref:web-api/overview/getting-started-with-aspnet-web-api/tutorial-your-first-web-api
+).
 
 For simplicity, the client application in this tutorial is a Windows console application. **HttpClient** is also supported for Windows Phone and Windows Store apps. For more information, see [Writing Web API Client Code for Multiple Platforms Using Portable Libraries](https://blogs.msdn.com/b/webdev/archive/2013/07/19/writing-web-api-client-code-for-multiple-platforms-using-portable-libraries.aspx)
 
 <a id="CreateConsoleApp"></a>
 ## Create the Console Application
 
-In Visual Studio, create a new Windows console application and paste in the following code.
+In Visual Studio, create a new Windows console app named **HttpClientSample** and paste in the following code:
+
+[!code-csharp[Main](calling-a-web-api-from-a-net-client/sample/client/Program.cs?name=snippet_all)]
+
+The preceding code is the complete client app.
 
-[!code-csharp[Main](calling-a-web-api-from-a-net-client/samples/sample1.cs)]
+`RunAsync` runs and blocks until it completes. Most **HttpClient** methods are async, because they perform network I/O. All of the async tasks are done inside `RunAsync`. In a console application, it's OK to block the main thread inside of `Main`. In a GUI application, you should **not** block the UI thread.
 
-This code provides the skeleton for the application. The `Main` `RunAsync` method and blocks until it completes. The reason for this approach is that most **HttpClient** methods are async, because they perform network I/O. All of the async tasks will be done inside `RunAsync`. In a console application, it's OK to block the main thread inside of `Main`. In a GUI application, you should never block the UI thread.
+[!code-csharp[Main](calling-a-web-api-from-a-net-client/sample/client/Program.cs?name=snippet_run)]
 
 <a id="InstallClientLib"></a>
 ## Install the Web API Client Libraries
 
 Use NuGet Package Manager to install the Web API Client Libraries package.
 
-From the **Tools** menu, select **Library Package Manager**, then select **Package Manager Console**. In the Package Manager Console window, type the following command:
+From the **Tools** menu, select **NuGet Package Manager** > **Package Manager Console**. In the Package Manager Console (PMC), type the following command:
+
+`Install-Package Microsoft.AspNet.WebApi.Client`
+
+The preceding command adds the following NuGet packages to the project:
+
+* Microsoft.AspNet.WebApi.Client
+* Newtonsoft.Json
 
-[!code-console[Main](calling-a-web-api-from-a-net-client/samples/sample2.cmd)]
+Json.NET is a popular high-performance JSON framework for .NET.
 
 <a id="AddModelClass"></a>
 ## Add a Model Class
 
-Add the following class to the application:
+Examine the Product class:
```

Comment:
`Product`
Custom code, language, and framework keywords are supposed to use backticks. Also there are a few with bold markings rather than backticks in the article.
I'll just leave this one comment for all of them.